numbat/src/parser.rs

```diff
@@ -2,29 +2,56 @@
 //!
 //! Grammar:
 //! ```txt
-//! statement       →   expression | variable_decl | function_decl | dimension_decl | unit_decl | procedure_call | module_import
+//! statement       →   variable_decl | function_decl | dimension_decl | decorator | unit_decl | module_import | procedure_call | expression
+//!
+//! variable_decl   →   "let" identifier ( ":" type_annotation ) ? "=" expression
+//! type_annotation →   boolean | string | dimension_expr
```

Comment:
I think this is wrong? A type annotation would not contain `true`/`false` or an actual string, but one of the two keywords `bool` or `str` (or a `dimension_expr`).

Reply:
Ooooh ok, my bad I went too fast on this part!